engine/runtime/src/main/java/org/enso/interpreter/epb/EpbParser.java

```diff
@@ -0,0 +1,76 @@
+package org.enso.interpreter.epb;
+
+import com.oracle.truffle.api.source.Source;
+
+/** A class containing helpers for creating and parsing EPB code */
+public class EpbParser {
+  private static final String separator = "#";
+
+  /** Lists all the languages supported in polyglot eval. */
+  public enum ForeignLanguage {
+    JS;
+
+    /** @return a Truffle language ID associated with this language */
+    public String toTruffleLanguage() {
+      return "js";
```

Comment:
This isn't very extensible. Wouldn't it be better to associate the truffle language tag as a field in the enum constant?